### PR TITLE
[TR] PIM-4551: move the file storage command

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                  | A
| ------------------ | ---
| Migration script?  | - 
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | -

### CODE REVIEW

File: src/Akeneo/Bundle/FileStorageBundle/Command/StoreFileCommand.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        $storageFsAlias = $input->getArgument('storage');
+        if (!$this->hasFileSystem($storageFsAlias)) {
+            $output->writeln(sprintf('<error>"%s" is not a valid filesystem.</error>', $storageFsAlias));
+
+            return 1;
+        }
+
+        $rawFile = new \SplFileInfo($filePath);
+        $storer = $this->getRawFileStorer();
+        $file = $storer->store($rawFile, $storageFsAlias);
```
Comment (phaseinducer) [inline src/Akeneo/Bundle/FileStorageBundle/Command/StoreFileCommand.php]: align ?

**Comment by phaseinducer** [comment]
GTM

**Comment by skeleton** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Pull request merged**